src/tls.rs

```diff
@@ -25,10 +25,12 @@ pub(crate) enum TlsConfigError {
     Io(io::Error),
     /// An Error parsing the Certificate
     CertParseError,
-    /// An Error parsing a Pkcs8 key
-    Pkcs8ParseError,
-    /// An Error parsing a Rsa key
-    RsaParseError,
```

Comment:
I couldn't come up with any other idea than this removal. So please let me know your thoughts about it.